ilastik/shell/gui/aboutDialog.py

```diff
@@ -62,6 +62,11 @@
         volumina contributors</a></li>
     </ul>
     </p>
+
+    <p>ilastik uses icons made by
+        <a href="https://www.gimp.org/">the GIMP team</a> under
+        <a href="http://creativecommons.org/licenses/by-sa/4.0/">CC-BY-SA-4.0</a> license.
+    </p>
```

Comment:
Maybe also mention the other icons license (even if public domain) there as well. Even if not required.